crates/vault-registry/src/lib.rs

```diff
@@ -766,10 +989,28 @@ impl<T: Config> Module<T> {
         Self::is_vault_below_threshold(&vault_id, <PremiumRedeemThreshold<T>>::get())
     }
 
+    /// check if the vault is below the liquidation threshold. In contrast to other thresholds,
+    /// this is checked as ratio of `collateral / (issued - to_be_redeemed)`.
     pub fn is_vault_below_liquidation_threshold(
         vault_id: &T::AccountId,
     ) -> Result<bool, DispatchError> {
-        Self::is_vault_below_threshold(&vault_id, <LiquidationCollateralThreshold<T>>::get())
+        let vault = Self::get_rich_vault_from_id(&vault_id)?;
+
+        // the current locked backing collateral by the vault
+        let collateral = Self::get_backing_collateral(vault_id)?;
+
+        // the currently issued tokens in PolkaBTC
+        let tokens = vault
+            .data
+            .issued_tokens
+            .checked_sub(&vault.data.to_be_redeemed_tokens)
+            .ok_or(Error::<T>::ArithmeticOverflow)?;
```

Comment:
```suggestion
            .ok_or(Error::<T>::ArithmeticUnderflow)?;
```

Reply:
Do we want to saturate this?

Reply:
No I don't think so, since it should always be true that `to_be_redeemed <= issued`